test/e2e/app-dir/app/app/error/global-error-boundary/page.js

```diff
@@ -5,7 +5,9 @@ import { useState } from 'react'
 export default function Page() {
   const [clicked, setClicked] = useState(false)
   if (clicked) {
-    throw new Error('this is a test')
+    const e = new Error('this is a test')
+    e.digest = 'CUSTOM_DIGEST'
```

Comment:
This is not checking for the right digest as it's a client-side error which generally would not have a digest related to it. The digest is added automatically for errors happening in server components, can you add a test for that instead?